sxt/multiexp/multiproduct_gpu/kernel.h

```diff
@@ -37,7 +37,8 @@ __global__ void multiproduct_kernel(typename Reducer::value_type* out,
                                     const unsigned* indexes,
                                     const block_computation_descriptor* block_descriptors) {
   using T = typename Reducer::value_type;
-  extern __shared__ T shared_data[];
```

Comment:
Please comment with a note about the workaround and reference the clang issue

Reply:
Good idea. Added.